content/boilerplates/notes/1.2.md

```diff
@@ -0,0 +1,67 @@
+
+## General
+
+- **Added** `traffic.sidecar.istio.io/includeInboundPorts` annotation to eliminate the need for service owner to declare `containerPort` in the deployment yaml file.  This will become the default in a future release.
+- **Introduced** IPv6 alpha support for Kubernetes clusters.
+
+## Traffic Management
+
+- **Improved** locality based routing in multicluster environments. (link to doc)
+- **Improved** outbound traffic policy in `ALLOW_ANY` mode (lin to doc). Traffic for unknown HTTP/HTTPS hosts on an existing port will be forwarded as is. Unknown traffic will be logged in Envoy access logs.
+- **Added** support for setting HTTP idle timeouts to upstream services.
+- **Improved** Sidecar support for NONE mode (without iptables) (link to doc).
+- **Added** ability to configure the DNS refresh rate (need a link) for sidecar Envoys, to reduce the load on the DNS servers.
+
+## Security
+
+- **Improved** Extended the lifetime of self-signed Citadel root certificates to 10 years.
+- **Added** Kubernetes health check prober rewrite per deployment via `sidecar.istio.io/rewriteAppHTTPProbers: "true"` in the `PodSpec` annotation. (link to doc)
+- **Added** support for configuring the secrete paths for Istio mutual TLS certificates.
+- **Added** support for [PKCS 8](https://en.wikipedia.org/wiki/PKCS_8) private keys for workloads, enabled by the field `pkcs8-keys` on Citadel.
+- **Added** jitter(need a link) when sending CSR requests for certification generation.
+- **Improved** JWT public key fetching logic to make it more resilient to network failure.
+- **Fixed** SAN field in workload certificates is set as `critical`. - need clear explanation
```

Comment:
```suggestion
- **Fixed** [SAN](https://tools.ietf.org/html/rfc5280#section-4.2.1.6) field in workload certificates is set as `critical`. This is required because the `subject` field is empty for Istio workload certificates.
```